libenkf/src/time_map.c

```diff
@@ -27,7 +27,7 @@
 
 #include <ert/ecl/ecl_sum.h>
 
-#include <ert/enkf/ert_log.h>
+#include <ert_log.h>
```

Comment:
No my friend - this is just too radical; we stick to traditional of deeply nested headers; i.e. the headers in `res_util` should be installed in: `$PREFIX/include/ert/res_util/xxx.h`.